app/assets/stylesheets/locks.scss

```diff
@@ -0,0 +1,25 @@
+.datetimepicker-container {
+  position: relative;
+}
+
+.lock-dropdown-menu {
+  margin-top: 1em;
+}
+
+.lock-times {
+  margin: .5em;
+}
+
+.lock-input {
+  border-radius: 4px;
+  border: 1px solid #ccc;
+  height: 34px;
+  padding: 6px;
+  vertical-align: bottom;
+  width: 405px;
```

Comment:
100% did not work ?

Reply:
No it didn't.. I played around with it for a bit and for some reason it was ignoring my right margin, so I just went with hard-coded px. I could dig into it a little more though if you prefer 100%.